src/main/java/wooteco/subway/admin/controller/ErrorController.java

```diff
@@ -0,0 +1,31 @@
+package wooteco.subway.admin.controller;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import wooteco.subway.admin.dto.ErrorResponse;
+import wooteco.subway.admin.exception.StartStationNotFoundException;
+
+import java.sql.SQLException;
+
+@RestControllerAdvice
+public class ErrorController {
+    @ExceptionHandler(SQLException.class)
+    public ResponseEntity<ErrorResponse> handleSQLException(SQLException e) {
+        final ErrorResponse response = ErrorResponse.of("SQL 에러입니다.");
+        return new ResponseEntity<>(response, HttpStatus.valueOf(500));
```

Comment:
`HttpStatus.INTERNAL_SERVER_ERROR`를 사용하면 될 것 같아요 :) 
```suggestion
        return new ResponseEntity<>(response, HttpStatus.INTERNAL_SERVER_ERROR);
```